pcdsdevices/tests/test_pseudopos.py

```diff
@@ -216,21 +229,34 @@ class MyLUTPositioner(LookupTablePositioner):
          [8, 300],
          [9, 400],
          ]
-    )
+    ) * signs
     column_names = ['real', 'pseudo']
     lut = MyLUTPositioner('', table=table, column_names=column_names,
                           name='lut')
 
-    np.testing.assert_allclose(lut.forward(60)[0], 2)
-    np.testing.assert_allclose(lut.inverse(7)[0], 200)
-    np.testing.assert_allclose(lut.inverse(1.5)[0], 55)
-
-    lut.move(100, wait=True)
-    np.testing.assert_allclose(lut.pseudo.position, 100)
-    np.testing.assert_allclose(lut.real.position, 6)
-
-    assert lut.real.limits == (0, 9)
-    assert lut.pseudo.limits == (40, 400)
+    np.testing.assert_allclose(lut.forward(60 * ps)[0], 2 * rs)
```

Comment:
how did I not know `np.testing` existed...

Reply:
You know, I didn't actually _read_ the functions I was adding the sign swaps to, I was only looking for numbers and figuring out whether they were the real positions or the pseudo positions, so I also didn't realize `np.testing` existed.

My test here is just using the pre-existing test from Ken but adding the cases where either side is decreasing instead of increasing.